### PR TITLE
Add anomaly detector2

### PR DESCRIPTION
**Pull request by robdmc**

### CODE REVIEW

File: ambition_utils/anomaly/models.py [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    # JSON blob to persist the state of the tdigest
+    blob = JSONField(default=dict, null=True)
+
+    # User defined percentiles below and above which an anomaly will happen
+    percentile_low = models.FloatField(null=True)
+    percentile_high = models.FloatField(null=True)
+
+    # These configure the accuracy/space tradeoff for the t-digest state
+    delta = models.FloatField(default=.01, null=True)
+    K = models.FloatField(default=25, null=True)
```
Comment (jaredlewis) [inline ambition_utils/anomaly/models.py]: Capital K is a weird field name, do you think we could use something more descriptive?

File: ambition_utils/anomaly/models.py [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+    # (for example, direct sql computation)
+    IS_INCREMENTAL = True
+    PERCENTILE_LOW_DEFAULT = 5
+    PERCENTILE_HIGH_DEFAULT = 95
+
+    class Meta:
+        abstract = True
+
+    # unique id to identify this anomaly detector
+    uid = models.CharField(max_length=256, unique=True)
```
Comment (jaredlewis) [inline ambition_utils/anomaly/models.py]: Could this be a uuid field? or are we treating this more like a name?

File: ambition_utils/tests/time_helper_tests.py [hunk 3]
```diff
@@ -23,6 +25,8 @@ def test_get_time_zones_as_tuple(self):
         """
         Makes sure the time zones are returned as tuples and that eastern is first
         """
+        print 'REABLE THESE TESTS'
```
Comment (jaredlewis) [inline ambition_utils/tests/time_helper_tests.py]: Prints

File: ambition_utils/sql.py [hunk 4]
```diff
@@ -0,0 +1,199 @@
+import os
+import inspect
+from collections import namedtuple
+from django.template import Template, Context
+from django.db.utils import ProgrammingError
+
+
+class SQLBase(object):
```
Comment (jaredlewis) [inline ambition_utils/sql.py]: Im guessing these are used somewhere outside of the project?

File: ambition_utils/anomaly/models.py (hunk 1 above)
Comment (robdmc) [reply ambition_utils/anomaly/models.py]: actually.  This is the best name for it.  It corresponds directly to a variable in the published paper describing the algorithm.

File: ambition_utils/anomaly/models.py (hunk 2 above)
Comment (robdmc) [reply ambition_utils/anomaly/models.py]: We are treating this like a name

Comment (robdmc) [reply ambition_utils/anomaly/models.py]: We are treating it like a name.  I can change it to something like `name` or `unique_name`, but I thought a `str` uid field would properly describe it's purpose.

File: ambition_utils/anomaly/models.py (hunk 1 above)
Comment (jaredlewis) [reply ambition_utils/anomaly/models.py]: I would still come look at this model and have no clue what that means

File: ambition_utils/anomaly/models.py (hunk 2 above)
Comment (jaredlewis) [reply ambition_utils/anomaly/models.py]: Yeah its fine to keep it

**Pull request merged**